src/web/package.json

```diff
@@ -1,5 +1,13 @@
 {
-  "dependencies": {
+"name": "@senecacdot/telescope-frontend",
```

Comment:
This is a small nitpick. Because we do not run `prettier` on these files we lack proper formatting. Could you format it so that it is properly indented? Thank you for PR! :)

Reply:
Thanks for the review, I fixed the indent in that line. Can you review my PR again? Happy to contribute to Telescope.

Reply:
@peterle93 can you do another commit and push again, to include this new change in your PR?  Also, if you run `pnpm install`, it may reformat this for you (`npm install` does, I can't remember if `pnpm` will).

Reply:
How come we don't run prettier in package.json files?

Reply:
Because it fights with `npm`, which also reformats it when it reads it.